### ISSUE TITLE
App crash on Create New project or Open Existing project (Mac OSX)

### ISSUE THREAD

**Issue by Kenoshen** [AtomicGameEngine/AtomicGameEngine#689]
1. Open app (on a Mac running OSX El Capitan 10.11.3)
2. Select "Create New"
3. Add a project name (selected Typescript for lang)
4. Press "Create"
5. Selected 2d module
7. Loads for a second
8. App crash with this message:
```
ui/frames/ProjectFrame - cannot read property 'path' of undefined - Line: 421
```

Same error occurs when I try to open the .atomic project file created from the previous steps.

**Comment by JoshEngebretson** [comment]
Hello, are you running the current downloadable version from http://atomicgameengine.com/download/

I am unable to replicate this behavior given the steps provided.  What project name and path are you using to create the project in?  Thanks.

**Comment by Kenoshen** [comment]
ProjectName: "PacMan"
ProjectPath: "/Users/mwingfield/Documents/AtomicProjects"

Using typescript btw.

**Comment by Kenoshen** [comment]
Also, I just tried creating another project, only changed the name, and it seems to work.

Could just be something on the first create project?

Also, both projects I picked the 2d configuration.

**Comment by Kenoshen** [comment]
[Broken-PacMan.zip](https://github.com/AtomicGameEngine/AtomicGameEngine/files/165910/Broken-PacMan.zip)

Here is a zip of the broken project that continues to produce the error described.

**Comment by JoshEngebretson** [comment]
Great, thanks, the project is empty which is odd.  I will look into how this might happen this afternoon.  Thanks for the report!

**Comment by Kenoshen** [comment]
Thank you for the snappy response!

**Comment by JoshEngebretson** [comment]
Closed via #690
